client/src/components/NavigationBar.vue

```diff
@@ -49,23 +34,19 @@ export default defineComponent({
       routes,
     };
   },
-  computed: {
-    cssVars() {
-      return {
-        '--bg-color': this.bgColor,
-        '--text-color': this.textColor,
-        '--height': `${this.height + 'px'}`,
-      };
-    },
-  },
 });
 </script>
 
-<style scoped>
+<style scoped lang="scss">
+@use '@blueconduit/copper/scss/01_settings/design-tokens' as dt;
```

Comment:
nit: just added these also and found the 'dt' prefix to be annoying / repetitive - can you import instead of use and remove it? 

if you disagree and think we should keep prefix I can add it, but we should be consistent

Reply:
SGTM and done.